lib/gems/pending/appliance_console/external_httpd_configuration.rb

```diff
@@ -15,6 +15,7 @@ module ExternalHttpdConfiguration
       SSSD_CONFIG          = "/etc/sssd/sssd.conf".freeze
       PAM_CONFIG           = "/etc/pam.d/httpd-auth".freeze
       HTTP_KEYTAB          = "/etc/http.keytab".freeze
+      HTTP_CONFIG_FILE     = "/etc/httpd/conf.d/manageiq-redirects-ws".freeze
```

Comment:
@abellotti 

I had originally coded the solution to check if evmserverd is running. However doing that turns out to be problematic because it can expose a timing issue, where evmserverd has been started but the http config file has not yet been created. Which results in the same traceback this PR attempting to avoid.